components/mealPlan/mealPlanController.js

```diff
@@ -113,6 +115,31 @@ module.exports.deleteMealPlan = function (req, res) {
 
         }
     });
+};
+
+module.exports.listMealPlans = function (req, res) {
+    var body = req.body;
```

Comment:
Hatte ich am Anfang auch auf GET. Problem sind dann die beiden Zeilen 
`router.route('/mealPlans/:id')
router.route('/mealPlans/list')`

Dadurch versucht das Backend nämlich, den String "list" in eine ObjectID umzuwandeln. Da ich die URL /mealPlans/list aber schön fand, war mein workaround die Methode auf POST umzustellen.

Alternativ könnte ich /mealPlans/list/:userId anbieten, glaube so ähnliche ist der getRecipefamilies Service und der hat auch keine Konflikte mit dem getRecipe. Hab ich aber gerade erst beim Review gesehen, finde ich eigentlich auch eleganter. 

Die User_ID im body (oder als URL param) würde ich weiterhin mit übergeben, finde ich vom Design sauberer, weil man dann theoretisch später nur ein Rechtesystem einbauen muss und danach ohne weitere Änderungen auch die MealPlans von anderen abfragen kann.